corehq/apps/cloudcare/static/cloudcare/js/formplayer/js/apps/menus/menu_controller.js

```diff
@@ -0,0 +1,69 @@
+/*global FormplayerFrontend */
+
+FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, FormplayerFrontend, Backbone, Marionette, $, _) {
+    MenuList.Controller = {
+        selectMenu: function (appId, stepList, page) {
+
+            var fetchingNextMenu = FormplayerFrontend.request("app:select:menus", appId, stepList, page);
+
+            /*
+             Determine the next screen to display.  Could be
+             a list of commands (modules and/or forms)
+             a list of entities (cases) and their details
+             */
+            $.when(fetchingNextMenu).done(function (menuResponse) {
+                var menuListView;
+                var menuData = {
+                    collection: menuResponse,
+                    title: menuResponse.title,
+                    headers: menuResponse.headers,
+                    widthHints: menuResponse.widthHints,
+                    action: menuResponse.action,
+                    pageCount: menuResponse.pageCount,
+                    currentPage: menuResponse.currentPage,
+                };
+                if (menuResponse.type === "commands") {
+                    menuListView = new MenuList.MenuListView(menuData);
+                    FormplayerFrontend.regions.main.show(menuListView.render());
```

Comment:
can be moved after the `if` statement since it's repeated. unless you expect there to be a `type` that isn't commands or entities